server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,306 @@
+/* Description: Ad defines the ad objects that we write to Firebase.
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
```

Comment:
Put the Javadoc on top of the class 'Ad' itself.

Nit: empty line between the package and the imports.